scripts/build.sh

```diff
@@ -5,6 +5,7 @@ rm -rf build
 ./node_modules/.bin/ngc
 
 #Nx client side lib
+echo "Copying Nx libraries to build/packages/nx"
```

Comment:
There are other libraries.. maybe add a log to the very end that lists the generated dists?